src/elements/radio-button/radio-button-group/radio-button-group.visual.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { html, type TemplateResult } from 'lit';
+import { styleMap, type StyleInfo } from 'lit/directives/style-map.js';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualDiffFocus,
+} from '../../core/testing/private.js';
+
+import '../../card/card-badge.js';
+import '../../form-error.js';
+import '../../icon.js';
+import '../../radio-button.js';
+
+const cases = {
+  disabled: [false, true],
+  orientation: ['vertical', 'horizontal'],
+  size: ['m', 's'],
+};
+
+const suffixStyle: Readonly<StyleInfo> = {
+  display: 'flex',
+  alignItems: 'center',
+  marginInline: 'var(--sbb-spacing-fixed-2x)',
+};
+
+const suffixAndSubtext = (): TemplateResult => html`
+  <span slot="subtext">Subtext</span>
+  <span slot="suffix" style="margin-inline-start: auto;">
+    <span style=${styleMap(suffixStyle)}>
```

Comment:
this span potentially could be combined with the parent span. Is all the additional styling in the suffix slot really needed?